src/main/resources/messages.properties

```diff
@@ -65,24 +63,20 @@ privacy.information-we-collect.ul2.li3=The operating system on your computer and
 privacy.information-we-collect.ul2.li4=Date and time of your visit
 privacy.information-we-collect.ul2.li5=Pages you visited
 privacy.information-we-collect.ul2.li6=Address of the website that connected you to the Site (such as google.com or bing.com)
-
 privacy.how-we-use.title=How we use your information
 privacy.how-we-use.content.p1=We use your information for our business purposes and legitimate interests such as: 
 privacy.how-we-use.content.li1=To help you obtain and keep benefits in Louisiana, including:
 privacy.how-we-use.content.li1.sub1=To complete forms required for the use of the Services
 privacy.how-we-use.content.li1.sub2=To submit your documents and personal information to the Louisiana Department of Children and Family Services
 privacy.how-we-use.content.li1.sub3=To access an account linked to your case in the Louisiana Department of Children and Family Services' online system
 privacy.how-we-use.content.li1.sub4=To send you messages related to your benefits application, updates, and renewals
-
```

Comment:
I turned on auto formatting (using the google stylesheet) and it causes this change.  Should I reverse this or do we really care?